### DISCUSSION TITLE
Variable  $kbd does not appear to be defined or passed into the button.index component.

### DISCUSSION THREAD

**Discussion by lcallif** [livewire/flux#158] [Feature Requests]
<?php if ($kbd): ?>
            <div class="text-xs text-zinc-500 dark:text-zinc-400">{{ $kbd }}</div>
        <?php endif; ?>

What is $kbd and where it is coming from ?

**Comment by idealerror** [comment]
It's signifying a keyboard shortcut.  Read the docs: https://fluxui.dev/components/button#as-an-input
